dist/stat_data.py

```diff
@@ -201,13 +201,17 @@ def __init__(self, name, desc, flags=''):
     ##########################################
     BackgroundCompactStat('background_compact_bytes_recovered', 'background compact recovered bytes', 'no_scale'),
     BackgroundCompactStat('background_compact_ema', 'background compact moving average of bytes rewritten', 'no_scale'),
-    BackgroundCompactStat('background_compact_exclude', 'background compact skipped file as it is part of the exclude list', 'no_scale'),
     BackgroundCompactStat('background_compact_fail', 'background compact failed calls', 'no_scale'),
     BackgroundCompactStat('background_compact_fail_cache_pressure', 'background compact failed calls due to cache pressure', 'no_scale'),
     BackgroundCompactStat('background_compact_files_tracked', 'number of files tracked by background compaction', 'no_scale'),
     BackgroundCompactStat('background_compact_interrupted', 'background compact interrupted', 'no_scale'),
     BackgroundCompactStat('background_compact_running', 'background compact running', 'no_scale'),
     BackgroundCompactStat('background_compact_skipped', 'background compact skipped file as not meeting requirements for compaction', 'no_scale'),
+    BackgroundCompactStat('background_compact_skipped_exclude', 'background compact skipped file as it is part of the exclude list', 'no_scale'),
+    BackgroundCompactStat('background_compact_skipped_missing_permissions', 'background compact skipped as there is a permissions issue', 'no_scale'),
+    BackgroundCompactStat('background_compact_skipped_no_such_file', 'background compact skipped, no such file exists', 'no_scale'),
```

Comment:
This is the only one that does not have a description consistent with the other one. I don't mind which one we go for but it makes things easier to be consistent:
```suggestion
    BackgroundCompactStat('background_compact_skipped_no_such_file', 'background compact skipped as no such file exists', 'no_scale'),
```

Reply:
Ah nice catch - 70922ff7bd118b38684c9e3f3999688408f9c4c3